src/main/java/org/bananarama/crud/util/WeldingAdapter.java

```diff
@@ -190,6 +190,15 @@ public void close() throws IOException {
             public void close() throws IOException {
                 throw new UnsupportedOperationException("Not supported yet.");
             }
+            @Override
+            public DeleteOperation<T> all() {
+
```

Comment:
MINOR - It's true that Bananarama doesn't have a style sheet, but I'd rather remove trailing and leading spaces in methods.